CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_marketo_source v0.7.2
+## Bug Fixes
+- Updated surrogate key `email_send_id` to include `primary_attribute_value_id`. The previous key was at a campaign level grain, not an email level grain. This is pertinent in the case where there are multiple emails that part of the same campaign. (https://github.com/fivetran/dbt_marketo_source/issues/25) 
+[#26](https://github.com/fivetran/dbt_marketo_source/pull/26)
+## Contributors
+- [sfc-gh-sugupta](https://github.com/sfc-gh-sugupta)
```

Comment:
Can you link the issue @sfc-gh-sugupta originally created. This was we can see their contribution since it is not a direct commit.

Reply:
It looks like this wasn't added. Would you be able to link the issue next to the account name